tests/container_test.rs

```diff
@@ -343,6 +343,44 @@ async fn attach_container_test(docker: Docker) -> Result<(), Error> {
     Ok(())
 }
 
+async fn resize_container_test(docker: Docker) -> Result<(), Error> {
+    create_shell_daemon(&docker, "integration_test_resize_container_tty").await?;
+
+    docker
+        .resize_container_tty(
+            "integration_test_resize_container_tty",
+            ResizeContainerTtyOptions {
+                width: 50,
+                height: 50,
+            },
+        )
+        .await?;
+
+    let _ = &docker
+        .kill_container(
+            "integration_test_resize_container_tty",
+            None::<KillContainerOptions<String>>,
+        )
+        .await?;
+
+    let _ = &docker
+        .wait_container(
+            "integration_test_resize_container_tty",
+            None::<WaitContainerOptions<String>>,
+        )
+        .try_collect::<Vec<_>>()
+        .await?;
+
+    let _ = &docker
+        .remove_container(
+            "integration_test_resize_container_tty",
+            None::<RemoveContainerOptions>,
+        )
+        .await?;
+
+    Ok(())
+}
+
```

Comment:
This test is a bit naive at the moment as it doesn't actually check that the TTY size changes. I might look into attaching to the container and using some commands to figure it out but not sure if it's worth it.